internals/generators/organization/config.yml.hbs

```diff
@@ -249,6 +248,11 @@ client:
         username:
           type: "email"
           pattern: '.+@.+\..+'
+          pattern: '.+@.+\..+'
+          label:
+            en: "email"
+          placeholder:
+            en: "enter email"
```

Comment:
I guess we can remove this since it's already present in the translations right?

Reply:
https://github.com/openwisp/openwisp-wifi-login-pages/pull/287#discussion_r671595861